webapp/src/errors.ts

```diff
@@ -79,6 +80,14 @@ function errorDefFromId(id: ErrorId | null): ErrorDef {
         errDef.button1Fill = true
         break
     }
+    case ErrorId.InvalidReadOnlyBoard: {
+        errDef.title = intl.formatMessage({id: 'error.invalid-read-only-board', defaultMessage: 'You don\’t have access to this board. Log in to access Boards.'})
+        errDef.button1Enabled = true
+        errDef.button1Text = intl.formatMessage({id: 'error.go-login', defaultMessage: 'Login'})
+        errDef.button1Redirect = window.location.origin + '/login'
```

Comment:
After login should they go to the board/card they were trying to access (like `ErrorId.NotLoggedIn`) ?

Reply:
... also is the `window.location.origin` needed, versus returning `/login` ?

Reply:
> After login should they go to the board/card they were trying to access (like `ErrorId.NotLoggedIn`) ?

Not sure, but my thought is no. The board they are going to is a read-only board (and may not exist or may be a bad (missing) token). We don't get back details about the error. Just that board wasn't retrieved. 

But, I can add it and try to send them back. I'll have to see how it works with an invalid board. I

Reply:
> ... also is the `window.location.origin` needed, versus returning `/login` ?

Yes, this was intention. `/login` will take them to `/plugins/focalboard/login`. So this was intentional. It is the same as what `ErrorId.TeamUndefined` does above. 

However, I need to test against Personal Server. (Forgot that one.)

Reply:
Works on Personal Server as well.